src/summaries/decentralized_exchanges.yaml

```diff
@@ -0,0 +1,128 @@
+  details: |
+
+    - Decentralized Exchange (DEX)
+      * A peer-to-peer marketplace where transactions between crypto traders are done mostly using Automated Market Maker (AMM) without banks, brokers, or any other intermediary intervention.
+      * Customer, transactional, and all other forms of data are stored on Blockchain. 
+      * Eg: Uniswap, dYdX (which runs on Ethereum Blockchain)
+
+      - Pros
+        * No KYC is required as trades are done in terms of account address 
+        * No fees are paid to the exchange.
+        * No impermanent loss (this will be discussed later in this course)
+        * Censorship resistance is governed by Smart Contracts and not by any central authority
+
+      - Cons
+        * Fees (i.e Gas Fee) are deducted as an additional charge for a deposit, withdrawal, or trade creation/cancelation.
+        * Trade settlement might be slow due to blockchain scalability issues. 
+        * Sometimes, not fully decentralized due to mediating server for speeding the process of trade execution.
+
+
+- title: Automated Market Maker (AMM)
+  shortTitle: AMM
+  key: amm
+  details: |
+    - Automated Market Maker (AMM):
+      * AMM is a type of DEX protocol that relies on a mathematical formula that allows digital assets to be traded without permission, automatically by using liquidity pools instead of the traditional order book pricing model. This formula can vary with each protocol. For example, Uniswap uses the Constant Product AMM protocol which is as follows:-
+        * Asset X quantity * Asset Y quantity = K (a constant)  simply  X*Y=K
+          * Properties of popular Constant Product AMM protocol:
+            * Instant liquidity, irrespective of the trade size.
+            * Purchase of asset X increases the price of X and decreases the price of Y
+            * The ratio of assets X and Y sets the price
+        * Now, What is a Protocol?
+          * Protocols are basic sets of rules that allow data and cryptocurrencies to be shared securely between different nodes (i.e computers) in the blockchain network.
+        * Then, What is Liquidity Pool?
+          * A Liquidity Pool is a collection of funds, funded by Liquidity Providers(LP) that are locked in a smart contract, to facilitate trading, borrowing, lending, yielding, etc.
+          * Types of rewards that Liquidity Providers get on funding Liquidity Pools are:
+            * Trading Fees: that are paid by traders on each trade( eg, 0.03% in Curve.fi )
+            - Liquidity Mining:
+              * It is an incentive/reward given to the Liquidity Providers(LP) proportional to the amount of liquidity provided by the LP
+              * It can also be provided to the LPs in terms of “Airdrops”.
+              * Airdrops are tokens that are given to the traders and LPs for free or in exchange for minimal promotional work.
+          - Types of losses for Liquidity Providers:
+            - Impermanent loss == Not Permanent Loss
```

Comment:
We should expand a bit more on slippage and impermanent loss since they are very important topics. We should try to have a few more questions also for them